src/services/getSingleProduct/getSingleProduct.jsx

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios'
+
+import { useNotificationContext } from '../../context/NotificationContext'
+import { useSingleProductContext } from '../../context/SingleProductContext'
+import { useGetProducts } from '../getProducts/getProducts'
+
+const useGetSingleProduct = ({ id }) => {
+  const { setNotificationOpen } = useNotificationContext()
+  const { setSingleProductData } = useSingleProductContext()
+
+  const { productsData } = useGetProducts()
+
+  const exact_id = productsData
```

Comment:
what is the difference between `id` and `exact_id`?
please use camelCase syntax